packages/babylon/src/parser/index.js

```diff
@@ -1,5 +1,6 @@
 /* @noflow */
 
+import uniq from "lodash/array/uniq";
```

Comment:
I would put this in babel-core around where I said since lodash isn't a dependency of babylon and it's already one in babel-core